modules/monitoring-getting-information-about-silences.adoc

```diff
@@ -15,7 +15,7 @@ You can find a silence and view its details.
 
 . Optional: Filter the silences by state using one or more of the state buttons *Active*, *Pending*, *Expired*.
 
-. Optional: Sort the silences by clicking one or more of the *Name*, *State*, and *Firing alerts* column headers.
+. Optional: Sort the silences by clicking one or more of the *Name*, *Firing alerts*, *State*, and *Creator* column headers.
```

Comment:
I guess this should be `Firing Alerts` to match the capitalization in the screenshot